### DISCUSSION TITLE
Distorted FM channel chirps

### DISCUSSION THREAD

**Discussion by GameTimeWarp** [frntc/Sidekick64#77] [General]
I'm getting odd FM sound out on "Rollin' with Tha Mouse [FM-YAM]" demo https://csdb.dk/release/?id=219225 ... The SID channel sounds fine but the FM channel is spitting out loud chirps. It should sound like https://www.youtube.com/watch?v=fselRVP2keg

Anybody else?

**Comment by frntc** [comment]
That should work. Only the currently published firmware does not include (the already existing) support for the 2nd FM digi channel (your other question). Will be included in the next release.

**Reply by GameTimeWarp** [reply]
Excellent! Looking forward to the update
